src/rmDefinitionAggregation.js

```diff
@@ -4,15 +4,15 @@ import axios from 'axios';
 const QueryURLTemplate =  "/recordm/recordm/definitions/search?"
 const ResultsURLTemplate = "#/definitions/__DEF_ID__/q=__QUERY__"
 
-const rmDefinitionAggregation = function (def, aggregation, query="*", from=0, size=10, sort="", ascending="asc") {
+const rmDefinitionAggregation = function (def, aggregation, query="*", from=0, size=10, sort="", ascending="asc", timezone="") {
```

Comment:
Is there a reason for the timezone to have default empty string ?

Reply:
Changed to `undefined`